File: WindowsApplication1/WindowsApplication1/Form1.vb
Public Class Form1

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim a, b, c, men As Integer
        a = Convert.ToInt32(TextBox1.Text)
        b = Convert.ToInt32(TextBox2.Text)
        c = Convert.ToInt32(TextBox3.Text)
etiqueta1:
        If a > b Then
            men = b
            b = a
            a = men
        End If
        If b > c Then
            men = c
            c = b
            b = men
            GoTo etiqueta1
        Else
            GoTo salir
        End If
salir:
        TextBox1.Text = Convert.ToString(a)
        TextBox2.Text = Convert.ToString(b)
        TextBox3.Text = Convert.ToString(c)
    End Sub
End Class

File: WindowsFormsApplication2/WindowsFormsApplication2/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication2
{

    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int a, b, c;
            a = Convert.ToInt32(textBox1.Text);
            b = Convert.ToInt32(textBox2.Text);
            c = a + b;
            textBox3.Text = c.ToString();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            Calculadora cal = new Calculadora();
            int a, b, c;
            a = Convert.ToInt32(textBox1.Text);
            b = Convert.ToInt32(textBox2.Text);
            c = cal.suma(a, b);
            textBox3.Text = c.ToString();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            calcu c = new calcu();
            c.Show();
        }


    }
}
